provisioning/roles/nginx/templates/drupal7-site.j2

```diff
@@ -0,0 +1,61 @@
+{% extends "default-site.j2" %}
+
+{% block index %}
+    index index.php;
+{% endblock %}
+
+{% block extra %}
+    {{ super() }}
+
+    # Seems that the "index" block is not injected from the parent template, add it again here
+    index index.php;
+
+    location = /favicon.ico {
+      log_not_found off;
+      access_log off;
+    }
+
+    location = /robots.txt {
+      allow all;
+      log_not_found off;
+      access_log off;
+    }
+
+    # This matters if you use Drush
+    location = /backup {
+      deny all;
+    }
+
+    location ~ \..*/.*\.php$ {
+      return 403;
+    }
+
+    location / {
+      # This is cool because no php is touched for static content
+      try_files $uri $uri/ @rewrite;
+    }
+
+    location @rewrite {
+        # You have 2 options here
+        # For D7 and above:
+        # Clean URLs are handled in drupal_environment_initialize().
+        rewrite ^ /index.php;
+        # For Drupal 6 and below:
+        # Some modules enforce no slash (/) at the end of the URL
+        # Else this rewrite block wouldn't be needed (GlobalRedirect)
+        # rewrite ^/(.*)$ /index.php?q=$1;
+    }
+
+    location ~ \.php$ {
+        fastcgi_pass unix:{{ fpm_socket }};
+        # Extra long timeout for running simpletests.
+        fastcgi_read_timeout 600;
+
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        include fastcgi_params;
+
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_intercept_errors on;
+    }
+
+{% endblock %}
```

Comment:
As a general comment, you could extend the php-config instead of the default one.